pkg/util/azureclient/insights/insights.go

```diff
@@ -0,0 +1,30 @@
+package insights
```

Comment:
this is all confused - I suggest not bothering creating pkg/util/azureclient/insights at all

Reply:
Its not part of main azure sdk. And still in development. This way I split it from production code (?)

Reply:
why not import github.com/Microsoft/ApplicationInsights-Go/appinsights direct when you need it? what's the purpose of this package?

Reply:
trying to be smart here :D not working very well :D